### PR TITLE
Update the bug template

### PR DESCRIPTION
**Pull request by kimwnasptd**
Update the bug template, based on o11y team.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+        Fetch the logs using `juju debug-log --replay` and `kubectl logs ...`. Additional details available in the juju docs
+        at https://juju.is/docs/olm/juju-logs
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
```
Comment (i-chvets) [inline .github/ISSUE_TEMPLATE/bug_report.yml:57]: Should be `Additional Context` as to be inline with other labels (capitalisation).

**Comment by kimwnasptd** [comment]
@i-chvets good points! Would you be OK if we merge this one with those discrepancies and then opening follow ups afterwards?

The reason is that `git-xargs` is giving me a hard time to force-push updates in the created branches. So to avoid deleting all the branches and recreating them can we
1. Merge the PRs with those small discrepancies
2. Open follow-up ones afterwards

This will reduce the manual work from my side to fetch all repos and push the updates

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (i-chvets) [reply .github/ISSUE_TEMPLATE/bug_report.yml:57]: ACK. Create follow-up on this.

**Comment by i-chvets** [comment]
ACK on creating follow up on capitalization.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        We need to know a bit more about the context in which you run the charm.
+        - Are you running Juju locally, on lxd, in multipass or on some other platform?
+        - What track and channel you deployed the charm from (ie. `latest/edge` or similar).
+        - Version of any applicable components, like the juju snap, the model controller, lxd, microk8s, and/or multipass.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
```
[the remark this answers is not in the document]
Comment (i-chvets) [reply .github/ISSUE_TEMPLATE/bug_report.yml:46]: ACK. Create follow-up on this.

**Review by i-chvets — APPROVED** [review]
Approved. ACK. Create follow-up on this.

**Comment by i-chvets** [comment]
Approved. Merge.

**Pull request merged**